### PR TITLE
MCKIN-30997 - In case of tie in user proficiency scores, sorting user position by modified date

### PR DESCRIPTION
**Pull request by ahmed-zubair12**
In case of tie in user proficiency scores, sorting user position by modified date as sorting leader board already use this criteria.

### CODE REVIEW

**Review by moeez96 — COMMENTED** [review]
Does this work for cases only with a tie or for all cases?

**Comment by ahmed-zubair12** [comment]
> Does this work for cases only with a tie or for all cases?

If there is a tie in score among users, then those users are further sorted on basis of `modified date`. So only in cases where there is a tie.

File: README.rst [hunk 1]
```diff
@@ -1,6 +1,9 @@
 gradebook-edx-platform-extensions
 ===================
 
+Notice: This repo will be archived in April 2021.
```
Comment (moeez96) [inline README.rst]: If you did not add this change lets rebase branch / remove it

**Comment by ahmed-zubair12** [comment]
Closing this PR. Creating a new PR using rebase-juniper branch.

**Pull request closed without merging**